decidim-comments/app/controllers/decidim/comments/comments_controller.rb

```diff
@@ -6,7 +6,9 @@ module Comments
     #
     class CommentsController < Decidim::Comments::ApplicationController
       include Decidim::ResourceHelper
+      include Decidim::SkipTimeoutable
 
+      prepend_before_action :skip_timeout, only: :index
```

Comment:
Can you move this to the `SkipTimeoutable` concern?

Reply:
How this is possible? Target varies, for example:

```
prepend_before_action :skip_timeout, only: :seconds_until_timeout
```

```
prepend_before_action :skip_timeout, only: :index
```

Can't be combined(?)

Reply:
Ah didn't notice, never mind!